compiler/optimizer/SinkStores.cpp

```diff
@@ -957,7 +908,7 @@ void TR_SinkStores::lookForSinkableStores()
                             store->_commonedLoadsUnderTree &&
                             store->_commonedLoadsUnderTree->get(killedSymIdx))
                            {
-                           traceMsg(comp(),"            store->containsUnsatisfedLoadFromSymbol(%d) = %d so %s temp\n",killedSymIdx,store->containsUnsatisfedLoadFromSymbol(killedSymIdx),store->containsUnsatisfedLoadFromSymbol(killedSymIdx) ? "needs":"doesn't need");
+                           traceMsg(comp(),"            store->containsUnsatisfedLoadFromSymbol(%d) = %d so %s temp\n",killedSymIdx, false, "doesn't need");
```

Comment:
I'm not sure if this trace message still makes sense, since the only thing that can change is `killedSymIdx`. Maybe it can be rephrased to something like

```c++
                           traceMsg(comp(),"            killedSymIdx = %d and temp is not needed\n",killedSymIdx);
```

Reply:
Agreed.  Fixed.